projects/wave-core/src/lib/operators/dialogs/data-repository/mapping-source.model.ts

```diff
@@ -102,6 +109,8 @@ export interface MappingSourceDict {
     };
     channels?: [MappingSourceRasterLayerDict];
     layer?: [MappingSourceVectorLayerDict];
+    time_start?: Time;
+    time_end?: Time;
```

Comment:
The source delivers strings and not `Time` types I guess. This is why you don't construct these types but just assign them. But in the end, `toString` of `Time` is not called. You see this in the different output format.

Reply:
fixed in 9919298afbd39ba8285c6946f1eec93e1c65b5ba